crmsh/utils.py

```diff
@@ -2675,12 +2679,12 @@ def calculate_quorate_status(expected_votes, actual_votes):
     return int(actual_votes)/int(expected_votes) > 0.5
 
 
-def get_stdout_or_raise_error(cmd, remote=None, success_val_list=[0], no_raise=False):
+def get_stdout_or_raise_error(cmd, user=userdir.getuser(), remote=None, success_val_list=[0], no_raise=False):
```

Comment:
In general, it looks to me to loop in SUDO_USER could help the usage with "sudo crm cluster init xxx", for example.

The code could leverage,
```
user=os.environ.get('SUDO_USER') or getpass.getuser()
```